EssentialEstablishmentGenerator/NPCGeneration/npcData.js

```diff
@@ -379,19 +910,21 @@ setup.npcData = {
       }
     },
     crime: {
-      probability: 5,
+      probability: 10,
       exclusions (town, npc) {
-        return true
+        if (setup.townData.professions[npc.profession].sector === 'crime') {

```

Comment:
amend to ` if (setup.townData.professions[npc.profession].sector === 'crime' || random(100) > 60) {` to give people that are not criminals the chance to commit crimes